src/controllers/favoriteController.js

```diff
@@ -0,0 +1,116 @@
+const userModel = require("../models/userSchema");
+
+/******************************************************************************
+ * @addFavoriteQuiz
+ * @route /api/v1/favorites/add
+ * @method POST
+ * @description add favorite quiz from the user in the favorites array in database
+ * @body quizId
+ * @returns favorites array
+ *******************************************************************************/
+
+//Add quiz as favorite
+const addFavoriteQuiz = async (req, res) => {
```

Comment:
I get a successful 200 response when I submit an empty body. It'd be good to have some validation to make sure that there is 1. a quizId in the body and 2. the quizId references an existing quiz.